content/pages/burials-and-memorials/application/530.md

```diff
@@ -10,7 +10,6 @@ layout: page-react.html
       <li><a href="/burials-and-memorials"> Burials and Memorials </a></li>
       <li><a href="/burials-and-memorials/survivor-and-dependent-benefits"> Survivors and Dependents </a></li>
       <li><a href="/burials-and-memorials/survivor-and-dependent-benefits/burial-costs/">Handling Burial Costs (Allowances)</a></li>
-      <li class="active">Apply for Burial Benefits</li>
```

Comment:
Turns out that the breadcrumbs partial renders different nav than the manually entered nav, so I'm going to leave it as-is for now.